appdatabase/migrations/sql/1687249080_add_position_accounts.up..sql

```diff
@@ -1,5 +1,76 @@
-ALTER TABLE keypairs_accounts ADD COLUMN position INT NOT NULL DEFAULT 0;
```

Comment:
Filename looks wrong, `..sql`

Reply:
but we can't change it, otherwise it will break existing migrated clients, nevermind